PENDING.md

```diff
@@ -3,6 +3,7 @@
 BREAKING CHANGES
 
 * Gaia REST API (`gaiacli advanced rest-server`)
```

Comment:
I'm not seeing the `PUT /stake/delegators/{delegators}/delegations` on the swagger file

Reply:
good catch. Updated